sdk/src/main/java/io/radar/sdk/Radar.kt

```diff
@@ -366,9 +361,9 @@ object Radar {
     }
 
     internal var initialized = false
-    private lateinit var context: Context
+    internal lateinit var context: Context
```

Comment:
why change these two from `private` to `internal`?

Reply:
ah, those were for [sending the conversion through the receiver](https://github.com/radarlabs/radar-sdk-android/pull/268#discussion_r1131523688), which got [removed](https://github.com/radarlabs/radar-sdk-android/pull/268#discussion_r1131628919). forgot to revert this change.